src/analysis/typepal/ConfigurableScopeGraph.rsc

```diff
@@ -9,8 +9,7 @@ import Map;
 import String;
 extend ParseTree;
 
-syntax ANONYMOUS_OCCURRENCE = "anonymous_occurence";
```

Comment:
That's not a real location. I'm not sure if the users of the location need it to be real.

Reply:
Otherwise why go through all this in the first place?

Reply:
Hmmm, I was under the impression that `[X]"string"` gave an "unkown" location.

Reply:
O that could be! But then why jump through all these hoops?

Reply:
ah, it gets the module loc, but is just `<0,length>`.